pkg/exporter/sumologicexporter/sender_test.go

```diff
@@ -1132,6 +1145,27 @@ func TestSendCompressZstd(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestSendCompressDeflate(t *testing.T) {
+	test := prepareSenderTest(t, configcompression.Deflate, []func(res http.ResponseWriter, req *http.Request){
+		func(res http.ResponseWriter, req *http.Request) {
+			res.WriteHeader(200)
+			if _, err := res.Write([]byte("")); err != nil {
+				res.WriteHeader(http.StatusInternalServerError)
+				assert.FailNow(t, "err: %v", err)
+				return
+			}
+			body := decodeZlib(t, req.Body)
+			assert.Equal(t, "deflate", req.Header.Get("Content-Encoding"))
```

Comment:
why is it `deflate`?

Reply:
That's the actual encoding the zlib format uses here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding.